docs/release-notes/release-notes-0.16.0.md

```diff
@@ -191,6 +191,11 @@ certain large transactions](https://github.com/lightningnetwork/lnd/pull/7100).
   using the address in question](
   https://github.com/lightningnetwork/lnd/pull/7025)
 
+## Pathfinding
```

Comment:
I usually squash this with the commit that contains the actual beef.

Reply:
makes sense, done!